packages/design-system-react/scripts/create-component/ComponentName/ComponentName.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { twMerge } from '../../utils/tw-merge';
```

Comment:
Will be available wen Text is merged https://github.com/MetaMask/metamask-design-system/pull/110